GestureHandler.web.js

```diff
@@ -0,0 +1,145 @@
+import React from 'react';
+import {
+  ScrollView,
+  Slider,
+  Switch,
+  TextInput,
+  ToolbarAndroid,
+  ViewPagerAndroid,
+  DrawerLayoutAndroid,
+  WebView,
+  FlatList,
+  TouchableWithoutFeedback,
+  findNodeHandle,
+} from 'react-native';
+import gestureHandlerRootHOC from './gestureHandlerRootHOC';
+import Directions from './Directions';
+import State from './State';
+
+// Factory for Handler components
+function createHandler(name, attachNativeHandler) {
+  return class Handler extends React.Component {
+    static displayName = name;
+
+    container = React.createRef();
+    detachNativeHandler = null;
+
+    componentDidMount() {
+      // Get current DOM node
+      const node = findNodeHandle(this.container.current);
+
+      // Attach handler to DOM node
+      const { enabled } = this.props;
+      if (node && attachNativeHandler && enabled !== false) {
+        this.detachNativeHandler = attachNativeHandler(node, this.props);
+      }
+    }
+
+    componentDidUpdate() {
+      // Get current DOM node
+      const node = findNodeHandle(this.container.current);
+
+      if (node && attachNativeHandler) {
+        // Detach existing handlers
+        if (this.detachNativeHandler) {
+          this.detachNativeHandler(node);
+        }
+
+        // Attach handler to DOM node again
+        const { enabled } = this.props;
+        if (enabled !== false) {
+          attachNativeHandler(node, this.props);
+        }
+      }
+    }
+
+    setNativeProps() {
+      // No implementation so far but is needed to avoid null calls
```

Comment:
You should be able to use `this.container.current.setNativeProps`

Reply:
I intentionally left this out since I think we do not need to pass on props for stub components. This would just generate warnings since most of the props are custom.